src/search/middleware/es-search.js

```diff
@@ -185,8 +185,8 @@ function getMatchQueries(query, { usePrefixSearch, fuzzy }) {
   const BOOST_EXPLICIT = 3.5
   // Number doesn't matter so much but just make sure it's
   // boosted low. Because we only really want this to come into
-  // play if nothing else matches. E.g. a search for `Acions`
-  // which wouldn't find anythig else anyway.
+  // play if nothing else matches. E.g. a search for `AcIons`
+  // which wouldn't find anything else anyway.
```

Comment:
Misspelled words are problematic. I'm proposing a different corruption of `Actions` one which based on a fairly standard programming convention can be read as two words  (reading `AcIons` as `Ac` + `Ions`, where two letter strings are ignored as too short to be checked against dictionary words and `Ions` is a word in the dictionary) but which still won't find anything (and makes it easier for a spell checker to complain about finding `anythig`).